zebra-chain/src/lib.rs

```diff
@@ -27,6 +27,16 @@ pub use redjubjub;
 #[cfg(test)]
 use proptest_derive::Arbitrary;
 
+use serde_big_array::big_array;
+
+/// The size of an Equihash solution in bytes (always 1344).
+const EQUIHASH_SOLUTION_SIZE: usize = 1344;
+
+big_array! {
+    BigArray;
+    +EQUIHASH_SOLUTION_SIZE, 580, 601, 80
```

Comment:
It might be helpful to explain which structs need arrays of size 580, 601, and 80.